src/pybind/mgr/dashboard_v2/frontend/src/app/shared/services/auth.service.ts

```diff
@@ -0,0 +1,18 @@
+import { Injectable } from '@angular/core';
+
+@Injectable()
+export class AuthService {
+
+  set(user) {
```

Comment:
Is user a string? Maybe we can use ``user: string`` here?

Reply:
Done